azurerm/internal/services/synapse/synapse_workspace_resource.go

```diff
@@ -146,6 +151,78 @@ func resourceSynapseWorkspace() *schema.Resource {
 				ValidateFunc: validate.ManagedResourceGroupName(),
 			},
 
+			"azure_devops_configuration": {
```

Comment:
I think we can drop configuration
```suggestion
			"azure_devops_source": {
```

or
```suggestion
			"azure_devops_repo": {
```

Reply:
`*_repo` sounds great!